csharp/src/sdk/Geometry/Color.cs

```diff
@@ -56,6 +84,33 @@ public Color(float red, float green, float blue, float alpha)
             this.Alpha = alpha;
         }
 
+        /// <summary>
+        /// Construct a color from an red, green, blue, and alpha value array.
+        /// </summary>
+        /// <param name="rgba">An array of the red, blue, green, and alpha color components between 0.0 and 1.0.</param>
+        public Color(float[] rgba)
```

Comment:
Pending the changes above, this constructor is no longer needed.

Reply:
Right, saw that, deleted.